clients/venice-admin-tool/src/main/java/com/linkedin/venice/datarecovery/Task.java

```diff
@@ -0,0 +1,72 @@
+package com.linkedin.venice.datarecovery;
+
+/**
+ * A Task is wrapper class that designed to execute multiple commands in sequence.
+ */
+public class Task implements Runnable {
```

Comment:
Just wanna share my feeling: I feel that these classes names, including Client, Module, Task, are too general. Are they going to be used besides data recovery?

Reply:
I was hoping the package name would help a bit to provide a namespace that's specific for data recovery, such that the Task should not be a general task, but a task that is specialized for data recovery. 
e.g. com.linkedin.venice.datarecovery.Task.
However, I can see where your feeling comes from. How about we give a prefix to these classes, like DataRecoveryTask, DataRecoevryModule etc?